keccak256/src/permutation/rho_checks.rs

```diff
@@ -118,74 +118,33 @@ use halo2_proofs::{
     plonk::{Advice, Column, ConstraintSystem, Error, Fixed, Selector},
     poly::Rotation,
 };
-use std::marker::PhantomData;
 
 #[derive(Debug, Clone)]
 pub struct LaneRotateConversionConfig<F> {
     q_normal: Selector,
     q_special: Selector,
     input_coef: Column<Advice>,
-    input_pob: Column<Fixed>,
-    input_acc: Column<Advice>,
     output_coef: Column<Advice>,
-    output_pob: Column<Fixed>,
-    output_acc: Column<Advice>,
     pub overflow_detector: Column<Advice>,
-    _marker: PhantomData<F>,
+    generic: GenericConfig<F>,
```

Comment:
That is really really nice! Makes things a lot easier!

Will rebase this into the final keccak PR ASAP!